src/main/resources/bidder-config/generic.yaml

```diff
@@ -4,6 +4,17 @@ adapters:
     aliases:
       genericAlias:
         enabled: false
+      blue:
+        enabled: false
+        endpoint: https://prebid-us-east-1.getblue.io/?src=prebid
+        meta-info:
+          maintainer-email: prebid@getblue.io
+          site-media-types:
+            - banner
+          app-media-types:
+            - banner
+          supported-vendors:
+          vendor-id: 0
```

Comment:
@SerhiiNahornyi are we sure that they don't have a GVL id?

Reply:
According to https://iabeurope.eu/vendor-list-tcf-v2-0/ they have
Value is 620

Reply:
In go https://github.com/prebid/prebid-server/pull/2323/files#diff-2ca656a1264a75707e04fdda5158c8f7fb3bff8c4afa71c3bf2fc729cd9919c8
They also have gvlVendorID with value 620.
@ValeriiBarsuk Pls update